modules/packages/ArgumentParser.chpl

```diff
@@ -232,9 +292,11 @@ module ArgumentParser {
   pragma "no doc"
   class SubCommand : ArgumentHandler {
 
-    proc init(cmd:string) {
+    proc init(cmd:string, help:argumentHelp) {
       super.init();
       this._name=cmd;
+      this._help=help;
+      this._kind=argKind.subcommand;
```

Comment:
Arguably it would be better to pass these as named arguments in `super.init()` but I don't consider it a big deal. The pattern appears in a few other places.

Reply:
thanks for the tip, fixed these